.travis.yml

```diff
@@ -15,7 +15,7 @@ script:
   - scripts/update_docs.sh
 
 after_success:
-  - tox -e coveralls
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
That's how they say to do it.

![screen shot 2016-06-20 at 4 31 36 pm](https://cloud.githubusercontent.com/assets/192456/16209322/838e328e-3704-11e6-9091-33be5456d8b5.png)